ubtue_Brill.js

```diff
@@ -95,8 +92,25 @@ function postProcess(doc, item) {
 	} else {
 		item.date;
 	}
-  if (!item.itemType)
+	if (!item.itemType)
 			item.itemType = "journalArticle";
+	//scrape ORCID from website
+	let authorSectionEntries = doc.querySelectorAll('.text-subheading span:nth-child(2)');
+	for (let authorSectionEntry of authorSectionEntries) {
+		let authorInfo = authorSectionEntry.querySelector('.c-Button--link');
+		let orcidHref = authorSectionEntry.querySelector('.orcid');
+		if (authorInfo && orcidHref) {
+			let author = authorInfo.childNodes[0].textContent;
+			var orcid = orcidHref.textContent.replace(/.*(\d{4}-\d+-\d+-\d+x?)$/i, '$1');
```

Comment:
Maybe better declare `orcid` with `let` before the `for`-loop.

Reply:
change made